docs/index.md

```diff
@@ -33,17 +33,14 @@ better alternative.
 <!-- schema generated by tfplugindocs -->
 ## Schema
 
-### Required
-
-- `domain` (String) Your Auth0 domain name. It can also be sourced from the `AUTH0_DOMAIN` environment variable.
```

Comment:
Not sure why this was included in the PR. Perhaps a previous change did not run `make docs`?